src/status_im/subs/profile.cljs

```diff
@@ -325,6 +325,23 @@
  (fn [[profile ens-names port font-file] [_ avatar-opts]]
    (replace-multiaccount-image-uri profile ens-names port font-file avatar-opts)))
 
+(re-frame/reg-sub
+ :profile/profile-user-names
+ :<- [:profile/profile]
+ :<- [:ens/names]
+ (fn [[profile ens-names] _]
```

Comment:
better to use destructuring `[[{:keys [display-name preferred-name]} ens-names] _]`

Reply:
yup yup will do, thank you 🙏